mirage/views/osf-resource.ts

```diff
@@ -96,7 +104,13 @@ export function osfNestedResource<K extends keyof ModelRegistry>(
     }
 
     if (actions.includes('show')) {
-        server.get(detailPath, mirageRelatedModelName);
+        server.get(detailPath, function(schema, request) {
+            const data = schema[mirageParentModelName].find(request.params.parentID)[relationshipName].models.map(
```

Comment:
Looks like this will just pull the first related model ever time rather than the model being looked up.
Something like `.find(model => model.id === request.params.id)` should work?